.circleci/config.yml

```diff
@@ -52,13 +52,14 @@ executors:
     working_directory: ~/app
   builder:
     docker:
-      - image: circleci/node:10-buster-browsers
+      - image: circleci/node:12.16.1-buster-browsers
     working_directory: ~/app
 
 orbs:
   aws-cli: circleci/aws-cli@0.1.18
   kubernetes: circleci/kubernetes@0.10.1
   helm: circleci/helm@0.2.3
+  gradle: circleci/gradle@2.0.0
```

Comment:
does this just magically add gradle to all executors? It doesn't matter in your case but you could imagine in other scenarios you'd want to limit it to specific executors.

Reply:
yeah, it's to all executors as far as I understand it, much like the other orbs we have